daphne/src/roles.rs

```diff
@@ -773,3 +728,78 @@ pub trait DapHelper<'a, S>: DapAggregator<'a, S> {
         })
     }
 }
+
+fn check_part_batch(
```

Comment:
It wasn't clear to me what this function is actually doing based on the name, but I don't have a better name suggestion :-\  The previous code used `check_batch_param` to cover the VDAF config check. Was that the intent here?

Reply:
This will become slightly less cryptic when we change BatchParameter to PartialBatchSelector up the PR stack. Bascially:

1. check_part_batch: Checks if the aggregation parameter is valid and that the PartialBatchSelector indicated by the Leader AggregateInitializeReq matches the query type for the task.
2. check_batch: Checks if the aggregation parameter is valid and that the BatchSelector indicated by the Leader in its AggregateShareReq (resp. the Collector in its CollectReq) matches the query type for the task.

Thus the terminology will at least be consistent. Still, it can be improved. Not super critical though given that this is an internal API. How about a comment explaining what the thing does, either here or where it's called?